app/engine/use-game-engine.ts

```diff
@@ -74,6 +74,7 @@ function stateToGameEngine(
      */
     answeredBy,
     answers: state.answers.get(clueKey) ?? new Map<string, string>(),
+    numAnswered: state.numAnswered,
```

Comment:
The game engine did not give us `numAnswered` (the number of answered clues in the round) from the State, so I added code to thread it through.